data.js

```diff
@@ -0,0 +1,70 @@
+var CarLot = (function(carlot) {
+  // Defining private varible to store parsed JSON data
+  var data = {}
+  // Function to get JSON data and then call a function to build the HTML page
+  carlot.loadInventory = (function() {
+    var xhr = new XMLHttpRequest();
+    xhr.addEventListener("load", function() {
+        data = JSON.parse(xhr.responseText).cars;
+        carlot.buildHTML(data)
+        carlot.activateEvents()
+    })
+    xhr.open("GET", "inventory.json")
+    xhr.send()
+  })()
+  // Adding a property to the global variable that will return inventory data
+  carlot.getInventory = function() {
+    return data;
+  }
+  // Function to build the HTML page with JSON data returned via loadInventory
+  carlot.buildHTML = function(data) {
+    for (i in data) {
+      if (i % 3 == 0) {
```

Comment:
Thanks, I was trying to follow the instruction's requirement for rows of three cards and had to structure it this way to get the row divs in. It would certainly produce the same result (and more concise code) to leave row classes off.